lib/safer_rails_console/patches/sandbox/redis_readonly.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module SaferRailsConsole
+  module Patches
+    module Sandbox
+      module RedisReadonly
+        READ_COMMANDS = [
```

Comment:
I'm vaguely worried about this list falling out of date as new commands are added, but I think it's fine since we're whitelisting.

Reply:
Also a nit, can we alphabetize these?